pkg/controllers/machine/disruption/drift.go

```diff
@@ -78,3 +78,31 @@ func (d *Drift) Reconcile(ctx context.Context, _ *v1alpha5.Provisioner, machine
 	// Requeue after 5 minutes for the cache TTL
 	return reconcile.Result{RequeueAfter: 5 * time.Minute}, nil
 }
+
+// isDrifted will increase in scope to include more fields of the provisioner
```

Comment:
nit: You can probably remove this comment now to describe what it actually does now